nncf/common/graph/graph_matching.py

```diff
@@ -29,6 +30,11 @@ def powerset(iterable, min_r=1, max_r=None):
 
 
 class Expression:
+    """A base class for objects that represent an expression to be matched against a subgraph of a directed
```

Comment:
Please, format the docstring according to the Style Guide.

Reply:
Done